test/clt-tests/sharding/test-drop-sharding-table.rec

```diff
@@ -0,0 +1,82 @@
+––– input –––
+export INSTANCE=1
+––– output –––
+––– block: ./base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=2
+––– output –––
+––– block: ./base/flexible/start-searchd –––
+––– input –––
+CLUSTER_NAME=c
+––– output –––
+––– input –––
+mysql -h0 -P1306 -e "create cluster ${CLUSTER_NAME}"; echo $?;
+––– output –––
+0
+––– input –––
+for n in `seq 2 $INSTANCE`; do mysql -h0 -P${n}306 -e "join cluster ${CLUSTER_NAME} at '127.0.0.1:1312'"; done; echo $?;
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "create table ${CLUSTER_NAME}:tbl1(id bigint) shards=3 rf=1;"; echo $?;
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "DROP SHARDED TABLE ${CLUSTER_NAME}:tbl1;"; echo $?;
+––– output –––
+0
+––– input –––
+sleep 3
+––– output –––
+––– input –––
+mysql -h0 -P1306 -e "SHOW TABLES\G"
+––– output –––
+*************************** 1. row ***************************
+Table: _sharding_queue
+Type: rt
+*************************** 2. row ***************************
+Table: _sharding_state
+Type: rt
+*************************** 3. row ***************************
+Table: _sharding_table
+Type: rt
+––– input –––
+mysql -h0 -P2306 -e "select * from _sharding_queue order by id asc limit 1000;"
```

Comment:
This is hidden table in the future and internal, we should not rely on the way we store data there so we should not include any selects from this table for test purpose other than just for debugging, so it's better to remove it

Reply:
Fixed